chevah/brink
# Add support for running from Unicode install path.

Scope
=====

Allow brink to be used with Unicode install paths.


Changes
=======

Add required code to have brink execute from a unicode path.

As a drive by, I have f fixed the code coverage reporting.

As a drive by, I have updated brink-pip to use the global cache, rather than the REPO/cache directory.

The build dir is now always "build/" on all OSes.

-------

It looks like on Windows, the Python.exe is not working with Unicode path... so for Windows we might only fix it with Python3 migration.

```
buildslave@bs3n-win-2019-x64-125 MINGW64 ~/chevah/bs/brink-win-i18n/build/build-ț (unicode-path)
$ ./lib/python
ImportError: No module named site
```


How to try and test the changes
===============================

reviewers: @dumol 

FYI  See the change in paver.sh. If paver.conf has a non-empty BUILD_FOLDER, it will be used as the build dir.

This is also a FYI for the state for brink slaves.

Brink is the first repo that was added to GitHub actions. You can see the result at https://github.com/chevah/brink/pull/119/checks
